src/views/Popup/routes/Home.tsx

```diff
@@ -255,30 +270,143 @@ export default function Home() {
     <div className={styles.Home}>
       <WalletManager />
       <div className={styles.Balance}>
-        <h1>
-          {formatBalance(balance()?.arBalance)}
-          <span>AR</span>
-        </h1>
+        <p className={styles.Address}>
+          <button onClick={() => setShowQRCode(true)}>
+            <svg
```

Comment:
Please don't use svgs in the code. The icons in the firma are from "iconicicons", the ones that it is missing should be imported from a separate svg file.